go.mod

```diff
@@ -205,6 +205,10 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
+// Temporary replace until https://github.com/lightningnetwork/lnd/pull/8831 is
+// merged.
+replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
```

Comment:
since this is close to merge - should we not split this up into 2 PRs? 1) that updates sqldb and 2) that points to that one? so that we dont merge with this replacement. or is the plan just to do an immediate follow up removing the replace?

Reply:
Both works imo, I'm happy to split it up once we get all approves :)